client/src/main/resources/templates/RuntimeReporter/index.html

```diff
@@ -22,8 +22,20 @@
         <link rel="stylesheet" href="../css/bootstrap-table.min.css">
         <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
         <!--[if lt IE 9]><script src="../bower_components/html5shiv/dist/html5shiv.js"></script><script src="../bower_components/respond/dest/respond.min.js"></script><![endif]-->
-        <style type="text/css">.modal-dialog {width: 95%; min-height: 95%; padding: 0;}.modal-content {min-height: 100%; }
-        </style>
+        <style type="text/css">.modal-dialog {width: 95%; min-height: 95%; padding: 0;}.modal-content {min-height: 100%; } </style>
```

Comment:
Let's leave this out of this PR and fix it for real later ..  Yes, I'm aware I won't be able to see the full impact of your improvement in this PR without this change... I can use the old-school SeLion Report though. :)